source/sync/configure/sync-settings.txt

```diff
@@ -172,12 +202,98 @@ Performance and Storage
 
 Each queryable field adds additional metadata storage to your Atlas cluster and
 may lead to degraded write performance. You should have as few queryable fields
-as needed by your application. A good rule of thumb is to have at most 10
-queryable fields.
+as needed by your application, and scope them to the minimum number of 
+collections required.
+
+Many apps find a good balance between storage usage and query flexibility 
+with at most 10 queryable fields applying to any single collection. 
+For example, if you have 3 *global queryable fields* and 7 
+*collection queryable fields*, you have 10 queryable fields that apply to the 
+collection.
+
+If you have a field that you only want to query in one collection, but it 
+is configured as a *global queryable field*, this unnecessarily consumes 
+Atlas storage space. For example, if you have a ``user`` field in every 
+collection, but you only use it for Sync queries in one collection, scoping 
+that as a *collection queryable field* reduces storage requirements. Reducing
+the scope means that Sync does not have to maintain metadata for that field
+for the other collections where you are not querying on the ``user`` field.
+
+If you need to reduce storage usage or improve performance, you can remove 
+unneeded queryable fields from your App. However, be aware of the consequences 
```

Comment:
The 'consequences of adding/removing a queryable field here' only apply while sync is running. Would you mind making a note about this here?